kasa/smartcamera/modules/pantilt.py

```diff
@@ -0,0 +1,59 @@
+"""Implementation of time module."""
+
+from __future__ import annotations
+
+from ...feature import Feature
+from ..smartcameramodule import SmartCameraModule
+
+
+class PanTilt(SmartCameraModule):
+    """Implementation of device_local_time."""
+
+    REQUIRED_COMPONENT = "ptz"
+
+    def _initialize_features(self) -> None:
+        """Initialize features after the initial update."""
+        self._add_feature(
+            Feature(
+                self._device,
+                "pan",
+                "Pan",
+                container=self,
+                attribute_setter="do_pan",
```

Comment:
```suggestion
                attribute_setter="pan",
```
Perhaps? I'm not sure if having `do_` prefix is that useful for the method names here.

Reply:
Done